files/qsub.pl

```diff
@@ -563,16 +563,38 @@ sub parse_script
         last if $line !~ m/^\s*(#|$)/;
         # oset and eset on separate line, in case -e and -o are on same line,
         # mixed with otehr opts etc etc
-        foreach my $pbsopt (qw(e o N)) {
+        foreach my $pbsopt (qw(e j o N)) {
             my $opts = $map{$pbsopt} || [$pbsopt];
-            my $pat = '^\s*#PBS.*?\s-'.$pbsopt.'\s+\S';
+            my $pat = '^\s*#PBS.*?\s-'.$pbsopt.'\s+(\S*)\s*';
```

Comment:
`\S*` or `\S+`?

Reply:
I think both of them good.

Reply:
they are very different ;) `\S*\s*` matches anything, incl empty strings

Reply:
I meant in this situation, not in general. But you are right, the rigorous way changing `\S` should be `\S+` and not `\S*`.